Parsers/ASimRegistryEvent/Parsers/VimRegistryEventMicrosoftSecurityEvent.yaml

```diff
@@ -0,0 +1,284 @@
+Parser:
```

Comment:
Parameter: actorusername_has_any - Not enough data to perform two values default tests

registryvalue_has_any - Got no results at all after filtering. Filtered by value: @'Enabled'

registryvalue_has_any - Got no results at all after filtering. Filtered by value: @'Enabled',@'MatchAnyKeyword'

registryvalue_has_any - Unable to find values to perform has_any tests 

Only one value exists for parameter: dvchostname_has_any - validations for this parameter are partial

dvchostname_has_any - Not enough data to perform two values default tests

Reply:
done